integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -130,15 +120,16 @@
  * Tests related to Domain events logged by operator.
  * The tests checks for the following events in the domain name space.
  * DomainCreated, DomainChanged, DomainDeleted, DomainCompleted,
- * DomainFailed, NamespaceWatchingStarted, and NamespaceWatchingStopped.
+ * DomainFailed.
  * The tests creates the domain resource, modifies it, introduces some validation errors in the domain resource
  * and finally deletes it to generate all the domain related events.
  */
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Verify the Kubernetes events for domain lifecycle")
 @IntegrationTest
-class ItKubernetesEvents {
+class ItKubernetesDomainEvents {
 
+  private static Class<?> thisClass = new Object(){}.getClass();
```

Comment:
thisClass.getClass().getSimpleName() produces "Class" not the class name string. You can delete this line.

Reply:
done